java/graalvm/Portfile

```diff
@@ -19,35 +21,11 @@ long_description GraalVM is a universal virtual machine for running applications
                  Kotlin, and LLVM-based languages such as C and C++.
 homepage         https://www.graalvm.org/
 
-master_sites     https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${version}/
-    
-checksums        rmd160  0d6e88a4a9227ab7a515d84409b4f279d4a6a016 \
-                 sha256  6c43063148368dc537a58706fcbf332583371ebb32dfdb78017e6a80e139658b \
-                 size    356582812
-
-distname         ${name}-ce-java8-darwin-amd64-${version}
-worksrcdir       ${name}-ce-java8-${version}
-
-use_configure    no
-
-build {}
+livecheck.type   none
 
-# macOS Java tools expect to find Java virtual machines under /Library/Java/JavaVirtualMachines, which is not under ${prefix}.
-destroot.violate_mtree yes
-
-configure.cxx_stdlib libstdc++
-
-set target /Library/Java/JavaVirtualMachines/${name}
-set destroot_target ${destroot}${target}
-
-destroot {
-    xinstall -m 755 -d ${destroot_target}
-    copy ${worksrcpath}/Contents ${destroot_target}
+pre-configure {
+    ui_error "Please do not install this port since it has been replaced by 'openjdk8-graalvm'."
+    return -code error
```

Comment:
Use the [obsolete portgroup](https://guide.macports.org/#development.obsolete-portgroup) and it takes care of this for you.

Reply:
I found https://guide.macports.org/chunked/development.practices.html and followed the instructions there. I guess those need to be updated then.

Reply:
Ah, only now did I see that the method of using the obsolete PortGroup is mentioned below. Maybe remove the instructions for the other method?

Reply:
Ah, only now did I see that the method of using the obsolete PortGroup is mentioned below. Maybe remove the instructions for the other method?

Reply:
That's a reasonable idea. (Heck, feel free to submit a pull request to do that...)

Reply:
I've modified the merge request to use the obsolete PortGroup for the `graalvm` port.